.github/workflows/build_main_documentation.yml

```diff
@@ -0,0 +1,137 @@
+name: Build PR Documentation
+
+on:
+  workflow_call:
+    inputs:
+      commit_sha:
+        required: true
+        type: string
+      package:
+        required: true
+        type: string
+      path_to_doc:
+        type: string
+      notebook_folder:
+        type: string
+    secrets:
+      token:
+        required: true
+
+jobs:
```

Comment:
in transformers [build doc workkflow](https://github.com/huggingface/transformers/blob/master/.github/workflows/build_documentation.yml#L45-L53), there is a `job` about `cache`:
```yml
      - name: Loading cache.
        uses: actions/cache@v2
        id: cache
        with:
          path: ~/.cache/pip
          key: v1-test_build_doc
          restore-keys: |
            v1-test_build_doc-${{ hashFiles('setup.py') }}
            v1-test_build_doc
```

do we need them here as well? or add them as a `workflow.inputs`

Reply:
No, this is using the docker image now.